lib/engine/game/g_1850/step/price_protection.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+require_relative '../../g_1870/step/price_protection'
+
+module Engine
+  module Game
+    module G1850
+      module Step
+        class PriceProtection < G1870::Step::PriceProtection
+          def can_buy?(entity, bundle)
+            return unless bundle&.buyable
+            return unless bundle == price_protection
+
+            have_cert_room = if bundle.corporation.counts_for_limit
+                               @game.num_certs(entity, price_protecting: true) + bundle.num_shares <= @game.cert_limit
+                             else
+                               true # can price protect yellow/green/brown even if over cert limit
+                             end
+
+            entity.cash >= bundle.price && have_cert_room
+          end
```

Comment:
copied over the can_buy? method from 1870.
In 1870 if player previously sold they can't price protect. 
They can in 1850